src/hci.h

```diff
@@ -870,6 +874,16 @@ typedef enum hci_init_state{
 
 } hci_substate_t;
 
+#define HCI_VENDOR_READ_CMD 0xFC61
```

Comment:
You can add vendor specific commands in hci_cmd.h and hci_cmd.c. Please use prefix hci_realtek_ for them.

Reply:
OK.